### ISSUE TITLE
TOPK on VARCHAR column does not work as expected

### ISSUE THREAD

**Issue by vahidhashemian** [confluentinc/ksql#748]
When TopK is performed on a `VARCHAR` column the resulting array is not properly populated.

Example: (tested on [this commit](https://github.com/confluentinc/ksql/commit/1433d1b7646bc98ce47a134eb78a29f82abee4e4))

1. Create a simple topic `city` of city names
2. `create stream city_stream (city varchar) with (value_format='delimited', kafka_topic='city')`
3. `create stream keyed_city_stream with (value_format='json') as select 1 as key_col, city from city_stream;`
4. `select key_col, topk(city, 5) from keyed_city_stream group by key_col;`

While populating the topic `city` with values `Oakland`, `San Diego`, `Portland`, and `San Francisco`, step 4 results in:
```
1 | [Oakland, null, null, null, null]
1 | [null, null, null, null, San Diego]
1 | [null, null, null, null, San Diego]
1 | [null, null, null, null, San Francisco]
```

**Comment by satybald** [comment]
@vahidhashemian thanks for submitting the issue. Incorrect null behavior is fixed in PR #694, in particular, this [commit](https://github.com/confluentinc/ksql/pull/694/files#diff-563da1c5e47a08e3d832d0ec499c6b36R67). Current master should have all these changes.

**Comment by vahidhashemian** [comment]
@sayat, thanks for the pointer to the fix. I confirm the issue is fixed at that commit. I'll close this issue then.
